packages/compass-import-export/src/components/export-select-fields.tsx

```diff
@@ -35,18 +35,14 @@ const tableContainerStyles = css({
 });
 
 const checkboxContainerStyle = css({
-  display: 'flex',
-});
-
-const cellContainerStyles = css({
-  padding: `${spacing[1]}px ${spacing[2]}px`,
+  // display: 'flex',
 });
 
-const inputCellContainerStyles = css(cellContainerStyles, {
-  verticalAlign: 'middle',
+const inputCellContainerStyles = css({
+  // verticalAlign: 'middle',
 });
```

Comment:
Pls. feel free to ignore this: These styles don't seem to be providing any styles so maybe they could be cleaned up?

Reply:
Ah, no nice catch!